packages/components/src/Chip/components/ChipSuffix/Chip.Suffix.tsx

```diff
@@ -26,6 +46,13 @@ export function ChipSuffix({ children, className }: ChipSuffixProps) {
 
 export interface ChipSuffixProps extends PropsWithChildren {
   readonly className?: string;
+  readonly showBG?: boolean;
+  readonly onClick?: (
+    value: string | number | undefined,
```

Comment:
Should we follow Atlantis pattern and put props before the function? 

Is there any technical benefit to where props are located within the file or is it purely personal preference? Maybe this is something we can talk about in one of our tech strategy meetings or in a retro because I hate to say "this is the way it's always been done" without personally knowing why we always put props on top

Reply:
I think it's just personal preference. I like to put the props below the component when they grow large enough to no longer be nicely inlined. When I open a component file I like to see the component, not a bunch of props/types first. But again, that's just personal preference, I think I intended to move those back above after I was done but clearly forgot! :)

Reply:
That reason does make sense! Thanks for moving it, maybe for internal components it's not that big of a deal but something we could still chat about with the team.  Never a bad thing to revisit why we do things a certain way